### ISSUE TITLE
cue: len of open-ended list should not evaluate to null

### ISSUE THREAD

**Issue by shasderias** [cuelang/cue#81]
Following-up from Slack conversation.
(1)
```
args: [...string]
argsLen: len(args)
```
evaluates to
```
args: []
argsLen: null
```
Expected argsLen to be 0 instead, i.e. expected (1) to evaluate to:
```
args: []
argsLen: 0
```
Possibly a non-issue or a separate issue,
(2)
```
args: *[] | [...string]
argsLen: len(args)
```
also evaluates to
```
args: []
argsLen: null
```
In (2), also expected argsLen to be 0.

**Comment by shasderias** [comment]
Oops, just saw the commit. Closed by f40ecd9012a23cdbbbb3405d0eb2a38caefdca89 (Change-Id: I7925132f01ada119a039cdec7d2a8c891f76fdc3)?

**Comment by mpvl** [comment]
The documentation is right: it always returns int (which includes an error in CUE)

The bug is fixed as of v0.0.8  (commit f40ecd901).

**Comment by cueckoo** [comment]
This issue has been migrated to https://github.com/cue-lang/cue/issues/81.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
